source/includes/vector-search-intro.rst

```diff
@@ -0,0 +1,97 @@
+The ``$vectorSearch`` aggregation stage performs an *approximate nearest neighbor* search
+on a vector in the specified field. Your collection *must* have a
+defined Atlas Vector Search index before you can perform a vector search on your data.
+
+.. tip::
+
+   To obtain the sample dataset used in the following example, see :ref:`csharp-quickstart`.
+   To create the sample Atlas Vector Search index used in the following example, see
+   :atlas:`Create an Atlas Vector Search Index </atlas-vector-search/create-index>` in the
+   Atlas manual.
+
+To create a ``$vectorSearch`` pipeline stage, call the ``VectorSearch()`` method on a
+``PipelineStageDefinitionBuilder`` object. The ``VectorSearch()`` method accepts the
+following parameters:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - Parameter
+     - Description
+
+   * - ``field``
+     - The field to perform the vector search on.
+
+       **Data type**: ``Expression<Func<TInput, TField>>``
+
+   * - ``queryVector``
+     - The encoded vector that will be matched with values from the database.
+       Although the data type of this parameter is ``QueryVector``, you can also pass an
+       array of ``float`` values.
+       
+       **Data type**: `QueryVector <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.QueryVector.html>`__
+
+   * - ``limit``
+     - The maximum number of documents to return.
+   
+       **Data type**: {+int-data-type+}
+   
+   * - ``options``
+     - Configuration options for the vector search operation.
+    
+       **Data type**: `VectorSearchOptions<TDocument> <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.VectorSearchOptions-1.html>`__
+
+You can use the ``options`` parameter to configure your vector search operation. The
+``VectorSearchOptions`` class contains the following properties:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Property
+     - Description
+
+   * - ``Exact``
+     - Whether the vector search uses the exact nearest neighbor (ENN) algorithm.
+       If this property is set to ``false``, the vector search uses the approximate nearest
+       neighbor (ANN) algorithm. If this property is set to ``true``, the
+       ``NumberOfCandidates`` property must be ``null``.
+   
+       | **Data type**: {+bool-data-type+}
+       | **Default**: ``false``
+
+   * - ``Filter``
+     - Additional search criteria that the found documents must match.
+   
+       | **Data Type:** `FilterDefinition<TDocument> <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.FilterDefinition-1.html>`__
+       | **Default**: ``null``
+   
+   * - ``IndexName``
+     - The index to perform the vector search on.
+   
+       | **Data type**: {+string-data-type+}
+       | **Default**: ``"default"``
```

Comment:
Sorry I wasn't entirely clear:
The default value for the property is null (default "default" might mean that the actual value is "default" if unchanged).
When `IndexName == null`, "default" is sent to the server.

Reply:
I don't remember what exactly happens when the string "default" is sent to the server.
Maybe the default index on the server is called "default".